server/channels/api4/plugin.go

```diff
@@ -58,6 +58,10 @@ func uploadPlugin(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := r.ParseMultipartForm(MaximumPluginFileSize); err != nil {
+		if err.Error() == "http: request body too large" {
+			c.Err = model.NewAppError("uploadPlugin", "api.plugin.upload.file_too_large.app_error", nil, "", http.StatusRequestEntityTooLarge)
```

Comment:
Additionally, the error message should point the admin to the place where they can update the value, i.e. http://localhost:8065/admin_console/environment/file_storage.

Reply:
I agree that `MaximumPluginFileSize` is named misleadingly, as you explained. I could modify that name to something like `MaximumPluginMemory`.

For the modifying the error message, what would be the best way to retrieve the link to that admin console page  (since the domain can vary) and return a clickable link in the error message?

Reply:
`MaximumPluginMemory` sounds good to me. Maybe the short `MaxPluginMemory` is even enough. Could you also please add a comment to it.

Reply:
`ServiceSettings.SiteURL` contains the base URL. From that we can hard code the rest.